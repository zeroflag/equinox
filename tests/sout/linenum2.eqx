"testing error/1" .

: my-abort ( -- )
  "test" >p error 1 ;

2 3 + dup * . cr

my-abort


