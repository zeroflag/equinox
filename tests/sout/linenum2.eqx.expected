testing error/1 25 
Error occurred at line: 4
  002.  
  003.  : my-abort ( -- )
=>004.    "test" >p error 1 ;
  005.  
  006.  2 3 + dup * . cr

